pgmoon/socket.lua

```diff
@@ -42,20 +42,25 @@ do
           end
           return self.sock:settimeout(t)
         end,
-        sslhandshake = function(self, verify, opts)
+        sslhandshake = function(self, opts)
           if opts == nil then
             opts = { }
           end
           local ssl = require("ssl")
           local params = {
             mode = "client",
-            protocol = opts.ssl_version or "tlsv1",
-            key = opts.key,
-            certificate = opts.cert,
-            cafile = opts.cafile,
-            verify = verify and "peer" or "none",
-            options = "all"
+            protocol = "any",
```

Comment:
This is one of the changes I wasn't sure of. We had originally switched from `protocol = opts.ssl_version or "tlsv1"` to `protocol = opts.ssl_version or "any"`. The original author droped the `opts.ssl_version` and just stuck with `"any"` so that seemed most appropriate to me. I would appreciate some confirmation!